src/operator/subgraph/build_subgraph.cc

```diff
@@ -676,14 +605,30 @@ void CreateSubgraphNode(Graph* g,
 #endif
 }
 
+/*!
+ * \brief Adjust a set of nodes belonging to the same subgraph.
```

Comment:
Please elaborate the meaning of `adjust`. Maybe using an example is good for explanation.

Reply:
Added.